convert/src/vertex_to_bd/badgerdoc_format/annotation_converter.py

```diff
@@ -0,0 +1,108 @@
+from typing import Any, List, Optional, Tuple
+
+from ..models import LabelStudioModel, ModelItem, ResultItem
+from ..models.bd_annotation_model import BadgerdocAnnotation, Obj, Page, Size
+from ..models.bd_tokens_model import Page as BadgerdocTokensPage
+
+
+class AnnotationConverter:
+    def __init__(self):
+        pass
+
+    def convert(
+        self,
+        annotations: LabelStudioModel,
+        badgerdoc_tokens: BadgerdocTokensPage,
+    ):
+        model_items: List[ModelItem] = annotations.__root__
+        badgerdoc_annotations = BadgerdocAnnotation(
+            pages=[Page(page_num=1, size=Size(width=100, height=100))]
+        )
+        for model_item in model_items:
+            for annotation in model_item.annotations:
+                for item in annotation.result:
+                    self.process_labels(
+                        badgerdoc_annotations, item, badgerdoc_tokens
+                    )
+                for item in annotation.result:
+                    self.process_relations(badgerdoc_annotations, item)
+        return badgerdoc_annotations
+
```

Comment:
@serereg, can you take a look?

Reply:
I'm not sure, but probably all relations should be proceeded after all labels. This is because we mutate the same badgerdoc_annotations object. We will add more tests to check it later.